sections/03_categorize.md

```diff
@@ -636,25 +636,24 @@ learned by the model and whether or not a model is sufficiently interpretable to
 identify biases. We discuss the challenge of model interpretability more
 completely in the discussion section.
 
-###### Temporal Patient Trajectories
-
-Traditionally, physician training programs justified long training hours by
-citing increased continuity of care and learning by following the progression of
-a disease over time, despite the known consequences of decreased mental and
-quality of life [@doi:10.1016/j.socscimed.2003.08.016
-@doi:10.1016/S1072-7515(03)00097-8 @doi:10.1097/00000542-199004000-00024
-@doi:10.1016/S0277-9536(96)00227-4]. Yet, a common practice in EHR-based
-research is to take a point in time snapshot and convert patient data to a
-traditional vector for machine learning and statistical analysis. This results
-in significant signal losses as timing and order of events provide insight into
-a patient's disease and treatment. Efforts to account for the order of events
-have shown promise [@doi:10.1038/ncomms5022] but require exceedingly large
-patient sizes due to discrete combinatorial bucketing. Lasko et al.
-[@doi:10.1371/journal.pone.0066341] used autoencoders on longitudinal sequences
-of serum urine acid measurements to identify population subtypes. More recently,
-deep learning has shown promise working with both sequences (Convolutional
-Neural Networks) [@arxiv:1607.07519] and the incorporation of past and current
-state (Recurrent Neural Networks, Long Short Term Memory
+###### Applications of Deep Learning to Longitudinal Analysis
+
+Traditional longitudinal analysis of large population cohorts, that have been
+prospectively followed either from birth, or the onset of particular conditions,
+have yielded important insights into the development of disease, and the factors
+contributing to health status, for example the Farmingham Heart Study in the
+US, and the Avon Longitudinal Study of Parents and Children in the UK. Yet, a
```

Comment:
This can be broken into two sentences.  Something like:

> Longitudinal analysis follows a population either prospectively from birth or from the onset of particular conditions.  In large patient populations, longitudinal analyses such as the Farmingham Heart Study [ref] and the Avon Longitudinal Study of Parents and Children [ref] have yielded important insights into the development of disease and the factors contributing to health status.

I also suggest adding a reference for each of the two exemplary studies.